README.md

```diff
@@ -86,6 +97,30 @@ configuration Sample_xDnsServerZoneTransfer_TransferToAnyServer
 Sample_xDnsServerZoneTransfer_TransferToAnyServer -DnsZoneName 'demo.contoso.com' -TransferType 'Any'
 ```
 
+### Configuring a Primary Standalone DNS Zone
+
+```powershell
+configuration Sample_xDnsServerPrimaryZone
+{
+    param
+    (
+        [Parameter(Mandatory)]
+        [String]$ZoneName,
+        [Parameter()] [ValidateSet('None','NonsecureAndSecure')]
+        [String]$DynamicUpdate = 'None' 
+    )
+    
+    Import-DscResource -module xDnsServer
+    xDnsServerPrimaryZone addPrimaryZone
+    {
+        Ensure        = 'Present'                
+        Name          = $ZoneName
+        DynamicUpdate = $DynamicUpdate
```

Comment:
ZoneFile property is missing in this sample. It is marked as mandatory property in the schema :)